### PR TITLE
WiFi-1826. Add OVSDB params for RRM channel hop config

### PR DESCRIPTION
**Pull request by netexp-ravi**
Adding Noise floor and OBSS related params to the RRM Config
OVSDB table.

Signed-off-by: ravi vaishnav <ravi.vaishnav@netexperience.com>

### CODE REVIEW

File: feeds/wlan-ap/opensync/patches/36-add-channel-hop-config.patch [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
++                    "min": 0,
++                    "max": 1
++                }
++            },
++            "non_wifi_time": {
++                "type": {
++                    "key": {
++                        "type": "integer",
++                        "minInteger": 0,
++                        "maxInteger": 500
```
[the remark this answers is not in the document]
Comment (netexp-ravi) [reply feeds/wlan-ap/opensync/patches/36-add-channel-hop-config.patch:48]: UI has the range already defined and for some reason, they are differences between the non-wifi and noise-floor time.
I just followed the same. Let me know if this needs a change.

File: feeds/wlan-ap/opensync/patches/36-add-channel-hop-config.patch [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                     "max": 1
+                 }
++            },
++            "noise_floor_thresh": {
++                "type": {
++                    "key": {
++                        "type": "integer",
++                        "minInteger": -90,
++                        "maxInteger": -10
++                    },
```
[the remark this answers is not in the document]
Comment (netexp-ravi) [reply feeds/wlan-ap/opensync/patches/36-add-channel-hop-config.patch:16]: The UI has a range defined for -10 and -90. Let me know if this still needs to be updated to -100.

File: feeds/wlan-ap/opensync/patches/36-add-channel-hop-config.patch (hunk 1 above)
[the remark this answers is not in the document]
Comment (RickSommerville) [reply feeds/wlan-ap/opensync/patches/36-add-channel-hop-config.patch:48]: Please check with UI for why the ranges are different.  Integration time of 0 is not really sensible.

**Review by RickSommerville — APPROVED** [review]

**Comment by RickSommerville** [comment]
Merged 58572f81, thanks.

**Pull request closed without merging**